### PR TITLE
Task15 without extra

### PR DESCRIPTION
**Pull request by vhuck91**

### CODE REVIEW

File: 15-PB-conditionals-pt1/index.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+let myNum3 = 99;
+
+if ((myNum1 >= 50 && myNum1 <= 99) || (myNum2 >= 50 && myNum2 <= 99)) {
+    console.log("true");
+} else {
+    console.log("false");
+}
+
+// 2. Check if three given integer values are in the range 50 to 99. Print true if one or more of them are in the range.
+function check_three_nums(myNum1, myNum2, myNum3) {
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:15]: This is not a functioning function. x, y and z are undefined, it is not printing anything, the exercise is about using if statements.

File: 15-PB-conditionals-pt1/index.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+if (age >= 13 && age < 20) {
+    console.log(`${firstName} is a teenager`);
+}
+
+if (age >= 20 && age < 30) {
+    console.log(`${firstName} is a young adult`);
+} else {
+    console.log(`${firstName} is an adult`);
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:127]: Missing a } at the end. Calling the file gives a syntax error.

File: 15-PB-conditionals-pt1/index.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+let mySum = myNum1 + myNum2;
+
+if (mySum >= 50 && mySum <= 80) {
+    console.log(65);
+} else {
+    console.log(80);
+}
+
+//6. Check whether the sum of two integers is 8, or whether their difference is 8. If one of these is the case, print true.
+if ((mySubstraction === 8) || (myNum1 - myNum2 === 8)) {
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:67]: mySubstraction is not defined

File: 15-PB-conditionals-pt1/index.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+if (Math.max(a, b, c) === a) {
+    console.log(a);
+}
+if (Math.max(a, b, c) === b) {
+    console.log(b);
+}
+if (Math.max(a, b, c) === c) {
+    console.log(c);
+}
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:33]: Yes it works but I feel like using Math is like cheating here. Exercise is about learning logic and if statements. Here you don't even need the if statements when using Math. Try redoing it without Math.

File: 15-PB-conditionals-pt1/index.js [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+//3 a) Then change the values of the variables to see if your conditional still works.
+
+a = 30;
+b = 40;
+c = 20;
+if (b < c && c > a) {
+    console.log(`this is largest value ${c}`)
+} else {
+    console.log(`false`)
+}
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:44]: You are only checking if c is the biggest, you should check a and b too.

File: 15-PB-conditionals-pt1/index.js (hunk 3 above)
Comment (Ungoot) [reply 15-PB-conditionals-pt1/index.js:67]: Where is the sum?

File: 15-PB-conditionals-pt1/index.js (hunk 5 above)
Comment (martl) [reply 15-PB-conditionals-pt1/index.js:44]: make sure you keep the data types boolean!

File: 15-PB-conditionals-pt1/index.js [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+if ((c === 15) || (d === 15) || ((d + c) === 15)) {
+    console.log("true");
+}
+
+//8. Check whether one of two integers is a multiple of 7 or 11. If so, print true.
+let myNum4 = 49;
+let myNum5 = 10;
+
+if ((myNum4 % 7 === 0) || (myNum5 % 11 === 0)) {
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:82]: You should check both numbers for both 7 and 11

File: 15-PB-conditionals-pt1/index.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+    console.log((myNum1 - 19) * 2);
+} else {
+    console.log(myNum1 - 19);
+}
+
+//11. BONUS CHALLENGE: Make a variable for firstName and age and give each variable values. Create an if/else statement to check whether the person's age is less than 13. If so, print "firstName is a child". If the age is equal to or more than 13 and less than 20, print "firstName is a teenager". If the person's age is equal to and more than 20 and less than 30, then print "firstName is a young adult". If none of these conditions apply, print "firstName is a adult".
+let firstName = "Aria";
+let age = 82;
+
+if (age < 18) {
```
Comment (Ungoot) [inline 15-PB-conditionals-pt1/index.js:116]: 13

**Pull request closed without merging**